conbench/app/_plots.py

```diff
@@ -283,6 +391,8 @@ def time_series_plot(history, benchmark, run, height=250, width=1000):
         )
         spacer.outline_line_color = None
         spacers.append(spacer)
-        print(f"Ignore any MISSING_RENDERERS warning for id={spacer.id}")
+        # Note(JP): I cannot make sense of this warning. I think the printed
+        # text is not acted on by anyone. Outcommeting for now.
+        # print(f"Ignore any MISSING_RENDERERS warning for id={spacer.id}")
```

Comment:
I put this print line in originally because there's an expected bokeh MISSING_RENDERERS warning that's logged every time these empty spacer objects are rendered, and I didn't want it to be confused with any true warnings that might be thrown if a plot doesn't render in the webapp. The true way to fix it would be to either 1. figure out a better way to add a spacer, or 2) add a filter to the `bokeh` logger that filters out specific warning messages.

<img width="1105" alt="image" src="https://user-images.githubusercontent.com/16600275/211907388-2fde673a-d35d-4332-a97b-f25ee4631697.png">

Reply:
Thank you for clarifying!

Reply:
adding a commit that uses https://docs.bokeh.org/en/latest/docs/reference/layouts.html#spacer (removes original warning)

Reply:
Nice, this Spacer object looks like a great solution.